data/mi/municipalities/Lois-Richardson-5d143251-d27d-4cfe-a3dd-d9928fc5159a.yml

```diff
@@ -0,0 +1,15 @@
+id: ocd-person/5d143251-d27d-4cfe-a3dd-d9928fc5159a
+name: Lois Richardson
+given_name: Beth
+family_name: Bashert
```

Comment:
Ditto for checking this one.

Reply:
Good catch thanks!

I will fix in airtable and rerun the process. Are you finding these manually? Should I add a `lint_yaml` check for `name == given_name + family_name`?

Reply:
Yeah, I've gotten decent at eyeballing it. I would love a lint check, but middle names make it messy (plus, for the state legislative side, sometimes nicknames come into play like 'Brad "Rusty" XYZ'.) So it would have to be very carefully created.

Reply:
@csnardi 

I think Lois Richardson is the correct person. Resolved here:

https://github.com/openstates/people/blob/24e92efde165b1ce6d1563bdba03837d6da711e6/data/mi/municipalities/Lois-Richardson-5d143251-d27d-4cfe-a3dd-d9928fc5159a.yml

...re linting for "inconsistent names" I took at stab at that here:

https://github.com/resistbot/people/blob/aaa126de3719256a6547a091b670341fdb268001/scripts/lint_yaml.py#L401